cves/CVE-2016-8740.yml

```diff
@@ -222,3 +254,7 @@ mistakes:
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
   answer:
+    One of the biggest mistake is the lack of integration/unit tests
+    that verify for strange inputs and weird situations. In addition, better coding
+    practices like quality assurance testing would go a long way in resolving this
+    issue.
```

Comment:
Beyond testing they might have implemented better design to check input size. I agree that unit testing and QA would have helped to find the problem after its injection into the system.

Reply:
good catch